packages/-build-infra/src/stripped-build-plugins.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+
+const fs = require('fs');
+
+const FilterImports = requireBabelPlugin('babel-plugin-filter-imports');
+const FeatureFlags = requireBabelPlugin('babel-plugin-feature-flags');
+const StripClassCallCheck = requireBabelPlugin('babel6-plugin-strip-class-callcheck');
+const StripFilteredImports = requireBabelPlugin('./transforms/babel-plugin-remove-imports');
+const TransformBlockScoping = requireBabelPlugin('@babel/plugin-transform-block-scoping');
+const {
+  isInstrumentedBuild,
+  wantsEnabledFeatures,
+  getManuallyEnabledFeatures,
+} = require('./cli-flags');
+
+function uniqueAdd(obj, key, values) {
+  const a = (obj[key] = obj[key] || []);
+
+  for (let i = 0; i < values.length; i++) {
+    if (a.indexOf(values[i]) === -1) {
+      a.push(values[i]);
+    }
+  }
+}
+
+// ensures that a `baseDir` property is present on the babel plugins
```

Comment:
this comment isn't correct any more (since we moved to `require.resolve` instead of directly requiring)

Reply:
👍